src/Core/Components/InfoButton/FluentInfoButton.razor.cs

```diff
@@ -0,0 +1,77 @@
+using Microsoft.AspNetCore.Components;
+
+namespace Microsoft.FluentUI.AspNetCore.Components;
+
+public partial class FluentInfoButton
```

Comment:
This component must be accessible (you can use NVDA to validate it).
Example (playing with sound): 
- Enter to open and read the tooltip content
- Tab to navigate in the tooltip
- Escape to close the tooltip

https://github.com/microsoft/fluentui-blazor/assets/8350694/de068289-49fe-446d-a627-68678a1ee169

Reply:
I wanted to change from tooltip to popover, because it seems to me that the input management is better. I don't know how to display corner. How can I do it?

Reply:
You can't. At least not with the in-built capabilities. You would have to resort to CSS and JS to make it work which seems to me a bit overkill for something that is already there.

Reply:
How do I manage the focus on the tooltip?

Reply:
I don't know what to do. I ask for help.

Reply:
See https://theadminbar.com/accessibility-weekly/accessible-tooltips/

The tooltip itself does not have to be focusable

Reply:
keyboard events space and enter need to be handled. the tooltip with focus in react component works but i read on internet that it should not be done. what do you want to do?

Reply:
I'd stick to the article I linked to.

Reply:
Now is ok?

Reply:
No :-(
- Not accessible (Enter, ESC, Read and navigate tooltip content)
- Unit tests fail

Reply:
sorry for my distraction

Reply:
how to test? `aria-describedby` is created every time.

Reply:
I fixed the test and removed `AdditionalAttributes` because it gives an error:

`The property 'AdditionalAttributes' on component type 'Microsoft.FluentUI.AspNetCore.Components.FluentTooltip' cannot be set explicitly when also used to capture unmatched values. Unmatched values: role
`